src/transformers/models/donut/modeling_donut_swin.py

```diff
@@ -802,6 +802,18 @@ def _init_weights(self, module):
         elif isinstance(module, DonutSwinSelfAttention):
             init.zeros_(module.relative_position_bias_table)
 
+            coords_h = torch.arange(module.window_size[0])
```

Comment:
this is now written twice, here and in the init of the class which we probably want to create a separate func imo